index.html

```diff
@@ -0,0 +1,330 @@
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <section class="contact-form" id="contact">
+      <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <form action="https://formspree.io/f/meqvdpwn" method="post">
+          <ul>
+              <li>
+
+                  <input type="text" id="name" name="user_name" placeholder="First name" maxlength="30" required>
+              </li>
+              <li>
+
+                  <input type="email" id="mail" name="user_email" placeholder="Email addresss" required>
+              </li>
+              <li>
+
+                  <textarea name="user_msg" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+              </li>
+          </ul>
+          <button type="submit">Get in touch</button>
+      </form>
+      <div class="links footer">
+          <ul class="icons footer">
+              <li class="icons-list"><a href="#"><i class="fab fa-angellist icon-color"></i></a></li>
+              <li class="icons-list"> <a href="#"><i class="fab fa-github icon-color"></i></a></li>
+              <li class="icons-list"> <a href="#"><i class="fab fa-linkedin icon-color"></i></a></li>
+              <li class="icons-list"> <a href="#"><i class="fab fa-medium icon-color"></i></a></li>
+          </ul>
+
+      </div>
+      <div class="end-line"></div>
```

Comment:
- [x] You have done really well implementing portfolio design 🙂, but I noticed the social media icons below the form are not styled properly. It would be nice to style it to fit the figma design, this will make give your portfolio a more uniform look.
- [x] See below the image comparison.  


Figma Design
<img width="344" alt="Screenshot 2022-06-23 at 17 03 17" src="https://user-images.githubusercontent.com/82473943/175344190-48877fe5-1974-4000-8130-35e05abee6db.png">

Your Design 
<img width="382" alt="Screenshot 2022-06-23 at 16 59 00" src="https://user-images.githubusercontent.com/82473943/175343232-dd85ac52-ae78-4c4f-8cba-d0a75c472b60.png">